fastapi-app/services/ingredients.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import func, desc
+
+from models import Ingredient, recipe_ingredient
+
+
+def service_create_new_ingredient(ingredient, db):
+    new_ingredient = Ingredient(name=ingredient.name)
+
+    db.add(new_ingredient)
+    db.commit()
+
+    return new_ingredient
+
+
+def service_get_most_used_ingredients(number, db):
+    ingredients = db.query(Ingredient.name, func.count(
+                        recipe_ingredient.id.label('count'))
+                    ) \
```

Comment:
Never split code with \ as it's unreadable.. split on brackets. 
E.g. 
`
ingredients = db.query(
    Ingredient.name,
    func.count(
        recipe_ingredient.id.label('count')
    )
).join(
    recipe_ingredient,
    recipe_ingredient.ingredient == Ingredient.id
).group_by(Ingredient.name).order_by(
    desc(func.count(recipe_ingredient.id))
).limit(number)
`

Reply:
Github killed my formatting, I'll send it on slack 🥲